src/Projection/User/UserProjector.php

```diff
@@ -105,4 +106,28 @@ public function onTodoWasMarkedAsDone(TodoWasMarkedAsDone $event)
 
         $stmt->execute();
     }
+
+    /**
+     * @param TodoWasReopened $event
+     * @throws \RuntimeException if data of the the assigned user can not be found
+     */
+    public function onTodoWasReopened(TodoWasReopened $event)
+    {
+        $user = $this->userFinder->findUserOfTodo($event->todoId()->toString());
```

Comment:
all methods of the UserProjector look like the new one added by @basz 
So I'd say it is ok for the exercise but we should think about refactoring it.

Unfortunately, I cannot remember why assignee_id is not part of the events. However, this would be a better approach as an event should contain all data required to populate the read model. If we add the assignee_id to the events we can remove the UserFinder dependency and the exception if user cannot be found. It is task of the domain to protect invariants, so the user must exist if referenced in an event.